.gitignore

```diff
@@ -2,3 +2,4 @@ cscope*
 jiva-discussion
 build/bin/*
 vendor/*
+*.idea
```

Comment:
Just curious what are these files?

Reply:
The `.idea` directory is created when we open a project in IntelliJ. It contains project settings and configuration files.